test-framework/src/test/java/org/apache/brooklyn/test/framework/TestHttpCallTest.java

```diff
@@ -87,6 +96,11 @@ public void testHttpBodyAssertions() {
                 .configure(TestHttpCall.TARGET_URL, server.getUrl() + "/body.json")
                 .configure(TestHttpCall.TIMEOUT, new Duration(10L, TimeUnit.SECONDS))
                 .configure(TestSensor.ASSERTIONS, newAssertion("matches", ".*123.*")));
+        app.createAndManageChild(EntitySpec.create(TestHttpCall.class)
+                .configure(TestHttpCall.TARGET_URL, server.getUrl() + "/foo/bar")
+                .configure(TestHttpCall.TARGET_METHOD, TestHttpCall.HttpMethod.POST)
+                .configure(TestHttpCall.TIMEOUT, new Duration(10L, TimeUnit.SECONDS))
```

Comment:
Personal preference: we should use `Asserts.DEFAULT_LONG_TIMEOUT` rather than our code being littered with arbitrary timeouts. In my opinion, the only time one should set different timeouts for something we expect to not timeout is if it's a performance test.